src/calculateRentalCost.js

```diff
@@ -5,6 +5,22 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  const PRICE = 40;
+  const BASE_PRICE = days * PRICE;
```

Comment:
Consider using a more descriptive name for 'BASE_PRICE'. Since it's not the final base price after applying discounts, a name like 'totalPriceBeforeDiscounts' might be more accurate and descriptive.